test/test_fx_experimental.py

```diff
@@ -749,6 +751,97 @@ def test_subgraph_trivial_resnet(self):
         module_with_submodules = split_module(traced, m, lambda node: 0)
         module_with_submodules(a)
 
+    @skipIfNoTorchVision
+    def test_normalize_args(self):
+        m = resnet18()
+
+        class FunctionalTracer(torch.fx.Tracer):
+            def is_leaf_module(self, m: torch.nn.Module, module_qualified_name : str) -> bool:
+                leaves = set([torch.nn.BatchNorm2d])
```

Comment:
So you want to decompose all the modules except for BatchNorm?

Reply:
For testing purposes I want to trace everything down to the functional level, but BatchNorm2d is currently not traceable due to control flow content (see https://fb.quip.com/EgT5A3DFrpaJ#IbQACAPoQ0n)

Reply:
Might be worth adding a note.